frontend/src/App.vue

```diff
@@ -136,6 +138,8 @@ export default {
 </style>
 
 <style scoped>
+/* <v-footer> is transformed to <footer class="v-footer"> */
+/* eslint-disable-next-line vue-scoped-css/no-unused-selector */
```

Comment:
I hate that we need to disable the rule for a line. I'd rather remove this eslint rule. This gives us a false sense of security.